---
float64 line_angle_deg
float64 line_distance